src/interfaces/IPoolManager.sol

```diff
@@ -170,12 +170,16 @@ interface IPoolManager is IProtocolFees, IERC6909Claims, IExtsload, IExttload {
     /// @param to The address to mint the tokens to
     /// @param id The currency address to mint to ERC6909s, as a uint256
     /// @param amount The amount of currency to mint
+    /// @dev The id is converted to a uint160 to correspond to a currency address
+    /// If the upper 12 bytes are not 0, they will be 0-ed out
     function mint(address to, uint256 id, uint256 amount) external;
 
     /// @notice Called by the user to move value from ERC6909 balance
     /// @param from The address to burn the tokens from
     /// @param id The currency address to burn from ERC6909s, as a uint256
     /// @param amount The amount of currency to burn
+    /// @dev The id is converted to a uint160 to correspond to a currency address
+    /// If the upper 12 bytes are not 0, they will be 0-ed out
```

Comment:
This isnt currently true - `_mint` and `_burn` get called with `id` without 0-ing out.
Please can you pass in `id  & 0x00FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` to those functions. Or `uint160(id)`

Reply:
do you think it should be masked before accountDelta and _mint? 
`id &= 0x00FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF;`
or when calling _mint? it doesnt really matter since CurrencyLibrary.fromId(id) converts the uint256 to uint160

Reply:
none of them change the gas, but i think that looks the best!